frontend/components.js

```diff
@@ -484,24 +550,54 @@ const AboutOverlay = connect('displayAboutOverlay', actions)(
             'p',
             undefined,
             `The official Cardano team did not review this code and is not responsible for any damage
-          it may cause you. The CardanoLite project is in alpha stage and should be used for
-          penny-transactions only. We appreciate feedback, especially review of the crypto-related code.`
+        it may cause you. The CardanoLite project is in alpha stage and should be used for
+        penny-transactions only. We appreciate feedback, especially review of the crypto-related code.`
           ),
-          h('h4', undefined, ' CardanoLite is not a bank '),
+          h('h4', {class: 'header-margin'}, ' CardanoLite is not a bank '),
           h(
             'p',
             undefined,
             `
-          It does not really store your funds permanently - each
-          time you interact with it, you have to insert the mnemonic - the 12-words long root password
-          to your account. If you lose it, we cannot help you restore the funds.
-        `
+        It does not really store your funds permanently - each
+        time you interact with it, you have to insert the mnemonic - the 12-words long root password
+        to your account. If you lose it, we cannot help you restore the funds.
+      `
+          ),
+          h('p', undefined, 'Feedback and contributions are very welcome.'),
+          h(
+            'span',
+            {class: 'centered-row'},
+            h('input', {
+              type: 'checkbox',
+              checked: dontShowAgainCheckbox,
+              onClick: this.checkboxClick,
+              class: 'understand-checkbox',
+            }),
+            h(
+              'span',
```

Comment:
How about using `label` instead? It will change value of `input` if it will be wrapped in it. You can something like `onChange` on input then.